### ISSUE TITLE
CH playground - add run from selection

### ISSUE THREAD

**Issue by mikeTWC1984** [ClickHouse/ClickHouse#40457]
Please add "run from selection" option on /play. Currently it is tedious to deal with multiple SQL statements (need to comment or copy/paste one by one)

Also, if you run insert statements there is no any result or timestamp returned, so it's not clear if you ran it or not. Some indication would be useful. Another strange quirk with insert is that if you put comment after it it will fail:

![image](https://user-images.githubusercontent.com/31977106/185767887-44ee5da1-a8a5-4822-bd25-d3b1c171e8d7.png)

**Comment by DerekChia** [comment]
You should at least see the `Ok.` reply when inserting data. 

<img width="367" alt="image" src="https://user-images.githubusercontent.com/1457728/186388736-35f70825-0aed-4d39-bbf3-b274e139bc4a.png">

As the current playground does not support multi-statements, it is expecting another value to be inserted and that explains why it is expecting '('

`INSERT INTO clicks VALUES (11, now()) (12, now()) (13, now())`

**Comment by mikeTWC1984** [comment]
Yes it displays OK, but if you run insert multiple times page remain static. It feels like nothing happen.

It's actually single insert statement , just with comment. My expectation is that comments should be ignored. I think it's not playground specific though, same happens with jdbc driver.
I discovered because it because I was switching between select and  insert (commenting one or the other) , so if you add "run from selection" it won't be an issue.

**Comment by DerekChia** [comment]
> Yes it displays OK, but if you run insert multiple times page remain static. It feels like nothing happened.
> 
> It's actually single insert statement , just with comment. My expectation is that comments should be ignored. I think it's not playground specific though, same happens with jdbc driver. I discovered it because I was switching between select and insert (commenting one or the other) , so if you add "run from selection" it won't be an issue.

Agreed and have filed enhancement request to the playground repo.

**Comment by zhicwu** [comment]
> It's actually single insert statement , just with comment. My expectation is that comments should be ignored. I think it's not playground specific though, same happens with jdbc driver.

Hi @mikeTWC1984, just tried and all 3 cases work for me. Could you share the case not working for you so that I can enhance JDBC driver accordingly?

```sql
-- insert 1
insert into clicks values(11, now())


INSERT INTO clicks VALUES (11, now()) (12, now()) (13, now())


-- insert 1
insert into clicks values(11, now());

-- insert 2
```

Also I want to mention that the comment may not be ignored when `log_leading_comment` is enabled in JDBC driver, for example:

https://github.com/ClickHouse/clickhouse-jdbc/blob/42b1b2193931dc68b86cb5dfc86392d4c6d6601b/clickhouse-jdbc/src/test/java/com/clickhouse/jdbc/ClickHouseStatementTest.java#L130-L145

**Comment by mikeTWC1984** [comment]
@zhicwu It doesn't work when comment comes after insert. E.g.
```sql
insert into clicks values(11, now())
-- select 1
```

**Comment by zhicwu** [comment]
Thanks @mikeTWC1984. I thought it's a parsing issue in JDBC driver causing problem in server but it's the opposite. I'll think about adding an option in JDBC driver to discard comments in query.

**Comment by rienath** [comment]
<img width="598" height="67" alt="Image" src="https://github.com/user-attachments/assets/2db7b35c-1b71-45b9-b025-d7291885a6e3" />
